libqtile/config.py

```diff
@@ -83,18 +83,25 @@ class KeyChord:
     submappings:
         A list of Key or KeyChord declarations to bind in this chord.
     mode:
-        A string with vim like mode name. If it's set, the chord mode will
-        not be left after a keystroke (except for Esc which always leaves the
-        current chord/mode).
+        A string with vim like mode name. The name will be displayed in the
+        Chord widget when entered. By default, when the mode is set, the chord
+        mode will not be left after a keystroke (except for Esc which always
+        leaves the current chord/mode).
+    name:
+        A string to name the chord. The name will be displayed in the Chord
+        widget but, unlike "mode", the chord will be exited after the final
+        keystroke.
```

Comment:
How is the final keystroke "final"? What does this mean?

Reply:
It's the last key in the sequence e.g. if you have a chord that is `Mod+x` and `z` then `z` is the final keystroke.

Reply:
Gotcha... guess I need to read up on the key chord docs